### PR TITLE
Bl-1054-fixed the css for the case details

### PR DESCRIPTION
**Pull request by jacob-bro**
Found the problem with case details css and fixed it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "antd": "^4.18.3",
     "aws-sdk": "^2.1142.0",
     "axios": "^0.21.4",
+    "eslint": "^7.11.0",
```
Comment (Kobramall) [inline package.json:14]: Did eslint needed to be add? If so did you check with jake or someone first?

File: src/styles/css/styles.css [hunk 2]
```diff
@@ -810,7 +810,7 @@ td.ant-table-column-sort {
 .ant-card-body {
   background-color: #262626;
   border-radius: 50px 10px;
-  width: 400px;
+  width: fit-content;
```
Comment (Kobramall) [inline src/styles/css/styles.css:813]: Besides that looks good!

Comment (ramib5734) [inline src/styles/css/styles.css:813]: What did `400px` cause the code to do and how does `fit-content` fix that issue?

**Review by ramib5734 — CHANGES_REQUESTED** [review]
General question about contents of the code

File: package.json (hunk 1 above)
Comment (jacob-bro) [reply package.json:14]: I am checking with Ash. It was required to get the program to run for me. I will fix it if needed.

File: src/styles/css/styles.css (hunk 2 above)
Comment (jacob-bro) [reply src/styles/css/styles.css:813]: Sorry I should have explained. I have run this all by Ash and he said it is good. The card-body that is changed was the card the entire Case Details page was being rendered on. It was jamming it all into the small 400px width. By setting it to fit-content it is allowing it all to display properly full page and not all jammed to one side.

**Pull request merged**